media/css/firefox/channel.scss

```
@@ -0,0 +1,180 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+$font-path: '/media/fonts';
+$image-path: '/media/protocol/img';
+
+@import '../../protocol/css/includes/lib';
+@import '../../protocol/css/components/hero';
+@import '../../protocol/css/components/picto-card';
+@import '../protocol/components/sub-navigation';
+@import '../../protocol/css/components/newsletter-form';
+@import '../../protocol/css/components/call-out';
+@import '../../protocol/css/components/emphasis-box';
+
+//* -------------------------------------------------------------------------- */
+// Global
+
+.mzp-l-narrow {
+    max-width: $content-lg;
+}
+
+//* -------------------------------------------------------------------------- */
+// Hero
+.mzp-c-hero {
+    background-color: $color-gray-20;
+
+    .mzp-c-hero-desc {
+        margin: 0;
+    }
+}
+
+//* -------------------------------------------------------------------------- */
+// Picto Card
+
+.bugzilla .mzp-c-card-picto-content {
+    max-width: none;
+
+    &:before {
+        background: transparent url('/media/img/icons/warning.svg') no-repeat center;
+    }
+}
+
+//* -------------------------------------------------------------------------- */
+// Call-outs
+
+.mzp-c-call-out-compact {
+    background: transparent;
+
+    @media #{$mq-md} {
+        &.mzp-t-product-firefox,
+        &.mzp-t-product-beta,
+        &.mzp-t-product-developer,
+        &.mzp-t-product-nightly,
+        &.mzp-t-product-focus {
+            .mzp-c-call-out-content {
+                background-size: $layout-xl $layout-xl;
+                padding: 0 0 0 $layout-2xl;
```

This might need a bidi mixin to apply the same padding on TRL screens

```suggestion
                @include bidi(((padding, 0 0 0 $layout-2xl, 0 $layout-2xl 0 0),));
```